src/test/java/TestUtils.java

```diff
@@ -1,26 +1,19 @@
 import com.itextpdf.text.DocumentException;
 import com.itextpdf.text.pdf.*;
-import org.apache.commons.io.IOUtils;
 
 import java.io.*;
-import java.nio.charset.Charset;
-import java.util.ArrayList;
 
 public class TestUtils {
 
-    public static boolean checkDocumentIsFlat(InputStream fileFlattened) {
+    public static boolean checkDocumentIsFlat(InputStream fileFlattened) throws IOException, DocumentException {
         boolean isflat;
-        try {
-            PdfReader reader = new PdfReader(fileFlattened);
-            ByteArrayOutputStream bos = new ByteArrayOutputStream();
-            PdfStamper stamper = new PdfStamper(reader, bos);
-            AcroFields fields = stamper.getAcroFields();
-            isflat = fields.setField("name", "ChangedName");
-            return isflat;
-        } catch (IOException | DocumentException e) {
-            e.printStackTrace();
-            return false;
-        }
+        PdfReader reader = new PdfReader(fileFlattened);
```

Comment:
even if this is a testing util, all "Cleoseable" resources should be closed. In this case that would be `PdfReader` and `PdfStamper`. Use the `AutoCloseable` wrappers in conjunction with try-with-resources if necessary.

Reply:
I changed it to use try-with-resources statement.